### PR TITLE
Use the correct feature name for error message

### LINKED ISSUE #47280: Poor diagnostic for out discards in lang version 6
**Version Used**: VS 16.8 preview 2

**Steps to Reproduce**:

Set LangVersion to 6 and compile the following:

```csharp
public class C
{
    public void M(out int a)
    {
        a = 0;
        M(out _);
    }
}
```

**Expected Behavior**: error : Feature 'discard arguments' is not available in C# 6. Please use language version 7.0 or greater.

**Actual Behavior**: error CS8059: Feature 'tuples' is not available in C# 6. Please use language version 7.0 or greater.

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #47280

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Errors/MessageID.cs [hunk 1]
```diff
@@ -103,7 +103,7 @@ internal enum MessageID
         IDS_PathList = MessageBase + 12686,
         IDS_Text = MessageBase + 12687,
 
-        // available
+        IDS_FeatureDiscards = MessageBase + 12688
```
Comment (Youssef1313) [inline src/Compilers/CSharp/Portable/Errors/MessageID.cs:106]:
```suggestion
        IDS_FeatureDiscards = MessageBase + 12688,
```

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 7)

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs [hunk 2]
```diff
@@ -6565,9 +6565,9 @@ static void Ok()
 ";
             var comp = CreateCompilation(source, parseOptions: TestOptions.Regular6);
             comp.VerifyDiagnostics(
-                // (6,9): error CS8059: Feature 'tuples' is not available in C# 6. Please use language version 7.0 or greater.
+                // (6,9): error CS8059: Feature 'discards' is not available in C# 6. Please use language version 7.0 or greater.
                 //         _ = M();
```
Comment (cston) [inline src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs:6569]: >_ = M(); [](start = 27, length = 8)

This comment looks incorrect.

**Review by cston — APPROVED** [review]

**Comment by Youssef1313** [comment]
Many integration tests are failling:

- VerifyPreviousAndNextHistory
- AddAssemblyReferenceAndTypesToInteractive
- Backspace
- BackspaceBehavesLikeDelete

and MUCH MUCH more. This is **very unusual**. Can someone check what's going on before restarting the failing jobs?

**Comment by RikkiGibson** [comment]
My understanding is that once an integration test fails, the subsequent integration tests in a run do not have trustworthy results. /cc @sharwell

**Comment by Youssef1313** [comment]
@RikkiGibson Tests are failing in #47497 too.

**Comment by Youssef1313** [comment]
@RikkiGibson I think @sharwell is taking care of these integration tests failures. You can re-run the failed jobs and it should be ready to merge if integration tests passed.

**Comment by jcouv** [comment]
Merged/squashed. Thanks @Youssef1313 !

**Pull request merged**